runtime/jcl/uma/se9_exports.xml

```diff
@@ -20,12 +20,8 @@
 	SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 <exports group="se9">
-	<export name="Java_java_lang_Class_getNestHostImpl">
-		<include-if condition="spec.flags.opt_valhallaNestmates" />
-	</export>
-	<export name="Java_java_lang_Class_getNestMembersImpl">
-		<include-if condition="spec.flags.opt_valhallaNestmates" />
-	</export>
+	<export name="Java_java_lang_Class_getNestHostImpl"/>
+	<export name="Java_java_lang_Class_getNestMembersImpl"/>
```

Comment:
These should be moved to se11_exports.xml for inclusion only in Java 11 (or newer). You should expect link errors in earlier version because you can't export functions that don't exist.